cmd/kubeadm/app/master/manifests.go

```diff
@@ -282,15 +283,19 @@ func getComponentBaseCommand(component string) []string {
 	return []string{"kube-" + component}
 }
 
+func getCertName(certName string) string {
+	return path.Join(kubeadmapi.GlobalEnvParams.HostPKIPath, certName)
```

Comment:
So `path` isn't cross-platform? How strange, didn't know that!
But I would prefer a PR that does that only, because there are lots of them I think
Sounds good? We could open an issue for it